app.js

```diff
@@ -6,21 +6,26 @@ const {
   mkdirSync
 } = require('node:fs')
 const { join } = require('node:path')
+const process = require('node:process')
 const { format: urlFormat } = require('node:url')
 
 // External dependencies
 const bodyParser = require('body-parser')
 const sessionInCookie = require('client-sessions')
 const cookieParser = require('cookie-parser')
-const dotenv = require('dotenv')
 const express = require('express')
 const sessionInMemory = require('express-session')
 const nunjucks = require('nunjucks')
 
-// Run before other code to make sure variables from .env are available
-dotenv.config({
-  quiet: true
-})
+/**
+ * Add environment variables from .env file
+ * (Requires Node.js v20.12.0 or later)
+ *
+ * @see {@link https://nodejs.org/api/process.html#processloadenvfilepath}
+ */
+if ('loadEnvFile' in process) {
+  process.loadEnvFile()
+}
```

Comment:
Should we only do this from the `watch` command, so that it never happens on a remote host like Heroku/Railway/etc?

Reply:
Some teams have copied **.env** files into Docker containers for deployment

I'm already wary that this PR drops `.env` support for Node.js v20.9.0–20.11.1

```json
  "engines": {
    "node": "^20.9.0 || ^22.11.0"
  }
```

That might be a reason to put this change in v8

Reply:
I've moved this to the `gulpfile.js` now, to simplify the `app.js` and make sure it’s not used in production mode.